mobile/src/main/java/com/genymobile/roomconnect/roomengine/ConnectionListener.java

```diff
@@ -0,0 +1,9 @@
+package com.genymobile.roomconnect.roomengine;
+
+
+public interface ConnectionListener {
```

Comment:
The goal is for the ServiceModule to be connected and able to interact with its service (Slack for example). Some service may not required authentication. Even for slack there is no authentication, just a hardcoded token to use.